packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -83,40 +83,40 @@ The ChatAdapter and the ChatComposite are purpose-built to support single chat t
 
```

Comment:
It is hard to review the raw mdx so please see interfaces table in Adapters page of generated chromatic:
https://60c7ae6891f0e90039d7cd54-dcqzebkcqp.chromatic.com/?path=/docs/composite-adapters--page#interfaces